packages/database/src/modelClasses/Entity.js

```diff
@@ -152,6 +152,11 @@ export class EntityType extends DatabaseType {
     return this.parent_id ? this.model.findById(this.parent_id) : undefined;
   }
 
+  async getParent(hierarchyId) {
+    const ancestors = await this.getAncestors(hierarchyId, { generational_distance: 1 });
+    return ancestors && ancestors.length > 0 ? ancestors[0] : undefined;
+  }
```

Comment:
Hope the naming here isn't too clashing with the existing `parent()` method...
Tbh I wonder if we can remove that method as it seems a bit archaic...